server/log_rpc_server.go

```diff
@@ -252,6 +258,9 @@ func (t *TrillianLogRPCServer) GetInclusionProof(ctx context.Context, req *trill
 	if err != nil {
 		return nil, err
 	}
+	// Work out what percentage of the current log size this index corresponds to.
+	percent := float64(req.LeafIndex) / float64(root.TreeSize) * 100.0
```

Comment:
This looks like a booby trap for start of day operations on a new tree....

Reply:
I thought it couldn't get here if tree size was zero but I'll take another look.

Reply:
If there's already checks for that then fine -- I didn't look further up.